packages/stores/types.ts

```diff
@@ -744,6 +751,9 @@ type TUiStore = {
     setRealAccountSignupEnd: (status: boolean) => void;
     setPurchaseState: (index: number) => void;
     simple_duration_unit: string;
+    //TODO: remove when BE API is ready
```

Comment:
do we still need these observables?

Reply:
yes we need this observable 🙏 will remove the comment